### PR TITLE
[merged] Test that the atomic command is handling params properly

### PR DESCRIPTION
**Pull request by rhatdan**
Want to make sure that we do not regress the handling of test params.

### CODE REVIEW

File: tests/integration/test_params.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#          SECRET     (a generated sha256 hash inserted into test containers)
+
+# In addition, the test harness creates some images for use in testing.
+#   See tests/test-images/
+
+OUTPUT=$(/bin/true)
+
+rc=0
+# Test different values
+test jezec 'asdf jkl'
```
Comment (baude) [inline tests/integration/test_params.sh]: do you think we should be testing non-ascii characters?

Comment (rhatdan) [inline tests/integration/test_params.sh]: Yes  for internationalization.

Comment (adelton) [inline tests/integration/test_params.sh]: Could you make it into
```
test ježek 'asdf jkl'
```

then?

Comment (rhatdan) [inline tests/integration/test_params.sh]: Yes, but could you tell us the proper way to fix this.

**Comment by rhatdan** [comment]
@baude PTAL

**Comment by rhatdan** [comment]
Since this test is now passing with @baude latest patch I will merge.

**Comment by rhatdan** [comment]
@cgwalters-bot r+

**Pull request closed without merging**